packages/paste-core/components/display-heading/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+import {DisplayHeading} from '../src';
+
+describe('DisplayHeading', () => {
+  it('should render', () => {
+    const {getByText} = render(
+      <DisplayHeading as="h1" variant="displayHeading10">
+        test
+      </DisplayHeading>
+    );
+    expect(getByText('test')).toBeDefined();
+  });
+  it('should render as an h1 when passed as="h1', () => {
+    render(
+      <DisplayHeading as="h1" variant="displayHeading10">
+        test
+      </DisplayHeading>
+    );
+    const renderedDisplayHeading = screen.getByRole('heading', {level: 1});
+    expect(renderedDisplayHeading).toBeDefined();
+  });
+  it('should render as an h2 when passed as="h1', () => {
+    render(
+      <DisplayHeading as="h2" variant="displayHeading20">
+        test
+      </DisplayHeading>
+    );
+    const renderedDisplayHeading = screen.getByRole('heading', {level: 2});
+    expect(renderedDisplayHeading).toBeDefined();
+  });
+  it('should render as an h3 when passed as="h1', () => {
+    render(
+      <DisplayHeading as="h3" variant="displayHeading30">
+        test
+      </DisplayHeading>
+    );
+    const renderedDisplayHeading = screen.getByRole('heading', {level: 3});
+    expect(renderedDisplayHeading).toBeDefined();
+  });
+  it('should remove bottom margin when passed margin="space0', () => {
+    render(
+      <DisplayHeading as="h3" variant="displayHeading30" marginBottom="space0">
```

Comment:
Suggestion(non-blocking): to test the actual pixel values, you can wrap this test in `<Theme.Provider>`